nixjs-rt/src/builtins.ts

```diff
@@ -385,39 +387,75 @@ export function getBuiltins() {
     },
 
     isAttrs: (arg) => {
-      throw new Error("unimplemented");
+      if (arg instanceof Attrset) {
```

Comment:
Consider moving `_nixBoolFromJs` to `utils.js` and using it here:
```suggestion
      return nixBoolFromJs(arg instanceof Attrset);
```

Reply:
What about just `new NixBool(arg instanceof Attrset)`

Pushed it as that just then

Reply:
My reasoning is why have a `nixBoolFromJs` but not a `nixStringFromJs` and `nixNumberFromJs`, etc, it's much easier to just use constructor calls

Reply:
`nixBoolFromJs` is an optimisation. To avoid allocating new instances for every `true` and `false`.